installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3122,7 +3122,7 @@ INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL)
 
 /* Plugin developed using PluginEngine V1 */
 INSERT INTO pac_config_properties (`cfkey`,`value`,`application`,`profile`,`label`,`createdBy`,`createdDate`,`modifiedBy`,`modifiedDate`)
- VALUES ('plugins.in.v1','gcp,redhat,contrast,checkmarx,crowdstrike','job-scheduler','prd','latest','System',now(),NULL,NULL) ON DUPLICATE KEY UPDATE value = 'gcp,redhat,contrast,checkmarx,crowdstrike', modifiedBy='System', modifiedDate=now();
+ VALUES ('plugins.in.v1','gcp,redhat,contrast,checkmarx,crowdstrike,wiz','job-scheduler','prd','latest','System',now(),NULL,NULL) ON DUPLICATE KEY UPDATE value = 'gcp,redhat,contrast,checkmarx,crowdstrike,wiz', modifiedBy='System', modifiedDate=now();
```

Comment:
Where is rapid 7?

Reply:
added rapid7

Reply:
@plyubich Rapid7 is already added, only wiz is missing